git-edit-index

```diff
@@ -130,6 +131,8 @@ class IndexEntry(object):
             return cls('D', file)
         elif re.match(r'\?', status):
             return cls('?', file)
+        elif re.match(r'\!', status):
```

Comment:
I think the backslash is redundant here as `!` is not a special character, unlike `?` ([ref](https://docs.python.org/3/library/re.html#regular-expression-syntax)).